examples/scripts/python_api/systems/test_system.py

```diff
@@ -0,0 +1,31 @@
+from gz.math7 import Vector3d
+from gz.sim8 import Model, Link
+import random
+
+
+class TestSystem(object):
+    def __init__(self):
+        self.id = random.randint(1, 100)
+
+    def configure(self, entity, sdf, ecm, event_mgr):
```

Comment:
I think we could probably leave out the event manager for now as it is generally not used much and used async when it is.  The `async` parts concern me a bit with the interaction with python.

Reply:
My thinking is that if we ever create bindings for `SdfEntityCreater`, the `event_mgr` needs to be passed along. Currently, it's just an empty binding, so there's not much you can do with it in python other than passing it to some other C++ function. This is also the case for the ECM. 

If we don't include it now, adding it later will mean all existing scripts have to be updated.

Reply:
Ah, okay, in that case lets keep it.